microsoft-excel-online/1.0/modules/ROOT/pages/microsoft-excel-online-connector-reference.adoc

```diff
@@ -0,0 +1,4921 @@
+
+== Configurations
+---
+[[AuthorizationCodeConfig]]
+=== Authorization Code Config
+
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| <<AuthorizationCodeConfig_OauthAuthorizationCode, OAuth v2.0 Authorization Code>>
+ | The connection types to provide to this configuration. | | x
+| Response Timeout a| Number |  Response timeout for the request. The ones specified here will overwrite the default ones specified in the config. |  30 |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Response timeout unit for the request. The ones specified here will overwrite the default ones specified in the config. |  SECONDS |
+| Encoding a| String |  Canonical name of the encoding. See Oracle's supported encodings at https://docs.oracle.com/javase/7/docs/technotes/guides/intl/encoding.doc.html. |  |
+| Name a| String |  The identifier of this element used to reference it in other components |  | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  |
+|===
+
+==== Connection Types
+[[AuthorizationCodeConfig_OauthAuthorizationCode]]
+===== OAuth v2.0 Authorization Code
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Proxy configuration a| <<ProxyConfiguration>> |  Proxy configuration for the connector. |  |
+| Response Type a| String |  Specifies the method to use to send the resulting token back to your app. It must be `query`. |  query |
+| TLS configuration a| <<Tls>> |  Defines a configuration for TLS, which can be used from both the client and server sides to secure communication for the Mule app. If HTTPS is configured as the protocol, then, at a minimum, the user must configure the keystore configuration. |  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+| Consumer Key a| String |  OAuth consumerKey as registered with the service provider. |  | x
+| Consumer Secret a| String |  OAuth consumerSecret as registered with the service provider. |  | x
+| Authorization Url a| String |  The service provider's authorization endpoint URL. |  https://login.microsoftonline.com/{tenant}/oauth2/v2.0/authorize |
```

Comment:
Put plus signs around this URL and the next one so they are not clickable:
+https://login.microsoftonline.com/{tenant}/oauth2/v2.0/authorize+